caas/kubernetes/provider/ingress.go

```diff
@@ -122,9 +167,9 @@ func (k *kubernetesClient) listIngressResources(labels map[string]string) ([]net
 }
 
 func (k *kubernetesClient) deleteIngressResources(appName string) error {
```

Comment:
This is used internally when deleting a service but I'm wondering why it's NetworkingV1beta1() and not NetworkingV1()